ui.py

```diff
@@ -65,6 +65,15 @@ def get_read_value():
             print('Type \'read\' or \'not read\'')
 
 
+def confirm_read_status(book):
+    """ Displays message that tells users if they read the book or not.
+     :param books: the book list """
```

Comment:
Excellent to have a docstring, but the param for this method is `book` which is one book, not a list of books